app/support/auto_expire.rb

```diff
@@ -0,0 +1,33 @@
+class AutoExpire
```

Comment:
Can we call this `AutoExpireReservation` or something a little more descriptive?